server/plugin.go

```diff
@@ -83,7 +83,7 @@ func (p *Plugin) OnActivate() (err error) {
 
 	p.httpOut = httpout.NewService(p.conf)
 
-	p.store, err = store.MakeService(p.conf, p.httpOut)
+	p.store, err = store.MakeService(p.conf, p.API, p.httpOut)
```

Comment:
Don't we have the API already in the conf?

Reply:
Actually, the raw plugin KV API is missing there and for some reason I can't remember off the top of my head it was needed.